character-sheet.html

```
@@ -0,0 +1,562 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel='icon' href='./assets/favicon.ico' type='image/x-icon'/ >
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)